Fixed some Steering Wheel & Config Bug.

jasperxu opened the pull request:
1. Fixed Steering Wheel's Buttons Bug. 
When you press any key on the steering wheel, all keys are recognized as true.
Bit operations are using & instead of &&.

2. Fixed Steering Wheel is not Work.
In file CarPawnSimApi.cpp Line 90 we used the function getRCData() and checked the value of rc_data.is_initialized. However, the value of rc_data.is_initialized is not set in the function getRCData(). Causes rc_data.is_initialized to be in false all the time. So the value of rc_data.is_initialized is always false, and the steering wheel cannot be enabled.

3. Fixed RC failed to read in the configuration file.
At Line 546. The code is repeated. So we mast set RC like this.
```
"RC": {
    "RC": {
        "RemoteControlID": 0,
        "AllowAPIWhenDisconnected": false
    }
}
```

jasperxu committed:
Fixed Steering Wheel's Buttons Bug.
When you press any key on the steering wheel, all keys are recognized as true.
Bit operations are using & instead of &&.

jasperxu committed:
Fixed Steering Wheel is not Work.
In file CarPawnSimApi.cpp Line 90 we used the function getRCData() and checked the value of rc_data.is_initialized. However, the value of rc_data.is_initialized is not set in the function getRCData(). Causes rc_data.is_initialized to be in false all the time. So the value of rc_data.is_initialized is always false, and the steering wheel cannot be enabled.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/AirSim?pullRequest=1941) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/AirSim?pullRequest=1941) before we can accept your contribution.<br/><br/>:x: jasperxu [sign now](https://cla.opensource.microsoft.com/microsoft/AirSim?pullRequest=1941)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/AirSim?pullRequest=1941) it.</sub>

jasperxu committed:
Change Tab to Space.

jasperxu committed:
Fixed  RC failed to read in the configuration file.
At Line 546. The code is repeated. So we mast set RC like this.
```
"RC": {
    "RC": {
        "RemoteControlID": 0,
        "AllowAPIWhenDisconnected": false
    }
}
```

madratman commented:
Thanks for the fixes @jasperxu
